example/test/react_test_components.dart

```diff
@@ -282,9 +282,9 @@ class _NewContextProviderComponent extends react.Component2 {
   }
 
   render() {
-    Map provideMap = {'renderCount': this.state['renderCount']};
+    Map<String, dynamic> provideMap = {'renderCount': this.state['renderCount']};
 
-    Map complexValues = {
+    Map<String, dynamic> complexValues = {
```

Comment:
I'm not 100% sure for the reason this was changed, but the `<String, dynamic>` typing can be inferred:

```suggestion
    final complexValues = {
```

Reply:
https://github.com/cleandart/react-dart/pull/221/files#r342151687